File: Datalake/datalakeclient/src/@types/enums/SourceType.ts
export enum SourceType {
	Inopc = 0,
	Datalake = 1,
}

export function SourceTypeDescription(type: SourceType) {
	switch (type) {
		case SourceType.Inopc: return 'iNOPC Server'
		case SourceType.Datalake: return 'Datalake Node'
		default: return '?'
	}
}

File: NewTemp/Enums/SourceType.cs
using System.Text.Json.Serialization;

namespace DatalakeDatabase.Enums
{
	[JsonConverter(typeof(JsonStringEnumConverter))]
	public enum SourceType
	{
		Custom = -1,
		Inopc = 0,
		Datalake = 1,
	}
}
